jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -502,6 +528,10 @@ void MRoverArm::zero_position_callback(std::string channel, Signal msg) {
 }
 
 void MRoverArm::arm_adjust_callback(std::string channel, ArmAdjustments msg) {
+    if (control_state == ControlState::OFF) {
+        set_to_closed_loop();
+    }
+
```

Comment:
Honestly I don't think this is needed, and we can also remove the 4 lines right below this. We're about to call target_orientation, which should do both of these checks for us. If you disagree lmk